ModelBinders/GovUkMandatoryDateBinder.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.AspNetCore.Mvc.ModelBinding;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUkDesignSystem.Attributes.DataBinding;
+using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc.ModelBinding.Validation;
+
+namespace GovUkDesignSystem.ModelBinders
+{
+    /// <summary>
+    /// This model binder can be used to replace the default MVC model binder for a required date property. It will add
+    /// validation messages to the model state inline with the GovUk Design System guidelines.
+    /// This binder must be used alongside a GovUkDataBindingDateErrorTextAttribute attribute.
```

Comment:
I think this is the place to mention that it only works for dates supplied as three values called day/month/year

Reply:
done